src/components/schemas/org.yaml

```diff
@@ -16,3 +16,68 @@ org_out_embed:
       example: example.com
     currency:
       $ref: './fields.yaml#/currency'
+org_out:
+  type: object
+  additionalProperties: False
+  required:
+    - id
+    - name
+    - domain
+    - currency
+    - created_at
+    - updated_at
+    - is_verified
+    - bank_account_number
+    - bank_branch_ifsc
+    - corporate_credit_card_details
+  properties:
+    id:
+      $ref: './fields.yaml#/org_id'
+    name:
+      type: string
+      maxLength: 255
+      description: Represents the name of the org.
+      example: Demo Account
+    domain:
+      type: string
+      maxLength: 255
+      description: This field specifies the domain with which the organisation is registered.
+      example: example.com
+    currency:
+      $ref: './fields.yaml#/currency'
+    created_at:
+      $ref: ./fields.yaml#/created_at
+    updated_at:
+      $ref: ./fields.yaml#/updated_at
+    is_verified:
+      type: boolean
+      nullable: true
+      description: |
+        Determines whether this organization has been verified by a verifier.
+    bank_account_number:
```

Comment:
can we remove bank_account_number, bank_branch_ifsc and corporate_credit_card_details from this API ?